src/scripts.test.ts

```diff
@@ -611,9 +622,28 @@ info: Script "testing:testingCallingScript" successful`;
 
       assert.isTrue(output.indexOf(`MAIN NODE_APP_INSTANCE: tests`) > -1);
     });
+
+    it(`non test script -> not set to "tests"`, async () => {
+      delete process.env[globalConstants.envVariables.NODE_APP_INSTANCE];
+
+      const { output, isSuccess } = await withLogNodeInstance(`lint`, `--nc`);
+      assert.isTrue(isSuccess);
+
+      assert.isTrue(output.indexOf(`MAIN NODE_APP_INSTANCE: undefined`) > -1);
+    });
+
+    it(`non test script with "--testing"`, async () => {
+      delete process.env[globalConstants.envVariables.NODE_APP_INSTANCE];
+
+      const { output, isSuccess } = await withLogNodeInstance(`lint`, `--nc`, `--testing`);
+      assert.isTrue(isSuccess);
+
+      assert.isTrue(output.indexOf(`MAIN NODE_APP_INSTANCE: tests`) > -1);
+    });
   });
 
-  describe('Sonar-init script', () => {
+  // TODO To remove? Where is defined the "sonar-init" script?
```

Comment:
I skipped this because I do not understand how is the `sonar-init` script supposed to be defined. It is not in `package.json`.

Reply:
Let me fix this. I think I deleted the new script.